Resources/Prototypes/_CM14/Entities/Objects/IdCards/cm_id_medical.yml

```diff
@@ -0,0 +1,31 @@
+﻿- type: entity
+  parent: CMIDCardSilver
+  id: MarineCMOIDCard
+  name: chief medical officer ID card
+  components:
+  - type: PresetIdCard
+    job: MarineCMO
+
+- type: entity
+  parent: CMIDCardStandardDogtag
+  id: DogtagDoctor
+  name: doctor`s dogtags
```

Comment:
I couldn't confirm this, do doctors in cm actually wear dogtags instead of the low bronze level ID card?

That doesn't seem right. Even if they do, I'd rather they use the ID cards...

Reply:
as i know we dont have bronze cards.

Reply:
ah, we do.

Reply:
done